vllm/worker/model_runner.py

```diff
@@ -1065,3 +1170,15 @@ def _is_block_tables_empty(block_tables: Union[None, Dict]):
             value is None for value in block_tables.values()):
         return True
     return False
+
+
+def _move_extra_tensor_data_to_seq_outputs(
```

Comment:
Reorganizes the `sampled_extra_tensor_data` into sequence outputs in sampler output.